attributes/default.rb

```diff
@@ -18,4 +18,7 @@
 #
 
 default['build-essential']['compile_time'] = false
-default['build-essential']['msys']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys"
+default['build-essential']['mingw']['prefix'] = "#{ENV['SYSTEMDRIVE']}\\mingw"
+
+# Change the defaults on pre-requisite cookbooks.
+default['seven_zip']['syspath'] = true
```

Comment:
We were mucking with it inside the windows recipe and that seemed weird to me.  I can move it back there.  I don't really know what the convention is when it comes to downstream cookbooks overriding the default attributes of cookbooks they depend on...